src/Model/ModelManager.php

```diff
@@ -339,17 +339,10 @@ public function getIdentifierValues($entity)
                 continue;
             }
 
-            if (method_exists($value, '__toString')) {
-                $identifiers[] = (string) $value;
-
-                continue;
-            }
-
             $fieldType = $metadata->getTypeOfField($name);
             $type = $fieldType && Type::hasType($fieldType) ? Type::getType($fieldType) : null;
             if ($type) {
-                $identifiers[] = $type->convertToDatabaseValue($value, $platform);
-
+                $identifiers[] = 'binary' === $platform->getDoctrineTypeMapping($fieldType) ? (string) $type->convertToPHPValue($value, $platform) : $type->convertToDatabaseValue($value, $platform);
                 continue;
```

Comment:
Please, add a blank line before `continue` statement in order to respect **blank_line_before_statement** rule (I don't know why `flintci/pr` isn't failing with this violation).

Reply:
Done